tests/unit/checks/test_generic_legacy_conversion.py

```diff
@@ -320,7 +320,6 @@ def test_all_check_variables_present(fix_plugin_legacy):
         'ceph_osds_default_levels',
         'ceph_status_default_levels',
         'check_mk_only_from_default',
-        'checkpoint_connections_default_levels',
```

Comment:
This line must stay once you added this variable to cmk/utils/migrated_check_variables.py (see my comment above). Otherwise, a unit test will fail.